test/export/test_serialize.py

```diff
@@ -1222,16 +1223,115 @@ def forward(self, x):
         inputs = (torch.zeros(4, 4),)
         ep = export(f, inputs)
 
-        for node in ep.graph.nodes:
+        new_gm = copy.deepcopy(ep.graph_module)
+        new_gm.meta["custom"] = {}
+        new_gm.meta["custom"]["f"] = "bar"
+
+        for node in new_gm.graph.nodes:
             if node.op == "call_function" and node.target == torch.ops.aten.add.Tensor:
-                node.meta["quantization_tag"] = "foo"
+                node.meta["custom"] = {}
+                node.meta["custom"]["quantization_tag"] = "foo"
 
-        serialized_vals = serialize(ep)
-        ep = deserialize(serialized_vals)
+        new_ep = ep._update(new_gm, ep.graph_signature)
+        serialized_vals = serialize(new_ep)
+        new_ep = deserialize(serialized_vals)
 
-        for node in ep.graph.nodes:
+        self.assertEqual(new_ep.graph_module.meta["custom"]["f"], "bar")
+        counter = 0
+        for node in new_ep.graph.nodes:
+            if node.op == "call_function" and node.target == torch.ops.aten.add.Tensor:
+                counter += 1
+                self.assertTrue(node.meta["custom"]["quantization_tag"] == "foo")
+        self.assertEqual(counter, 1)
+
+    def test_custom_tag_metadata_decomp(self):
+        class Foo(torch.nn.Module):
+            def forward(self, x):
+                return x + x
```

Comment:
what is aten.add decomposed into?
Better test with an op that actually decomps.